src/edu/ucsb/cs56/projects/FTP_client/ClientGui.java

```diff
@@ -23,6 +23,7 @@
 	private JPanel loginPanel;
 	private JPanel downloadPanel;
 	private JButton connectButton;
+    private JButton cdButton;
```

Comment:
It looks like IntelliJ defaults to using spaces when the Tab key is pressed. I have changed my settings to use the "tab character" in place of spaces now.